File: app/src/main/java/com/example/mobile6/ui/appointment/AppointmentsFragment.java
package com.example.mobile6.ui.appointment;

import android.view.LayoutInflater;
import android.view.ViewGroup;

import com.example.mobile6.databinding.FragmentAppointmentsBinding;
import com.example.mobile6.ui.base.BaseFragment;

public class AppointmentsFragment extends BaseFragment<FragmentAppointmentsBinding> {


    @Override
    protected FragmentAppointmentsBinding inflateBinding(LayoutInflater layoutInflater, ViewGroup container) {
        return FragmentAppointmentsBinding.inflate(layoutInflater, container, false);
    }

    @Override
    protected void initViews() {
        super.initViews();
    }

    @Override
    public void onDestroyView() {
        super.onDestroyView();
    }
}

File: app/src/main/java/com/example/mobile6/ui/appointments/AppointmentsFragment.kt
package com.example.mobile6.ui.appointments

import android.view.LayoutInflater
import android.view.ViewGroup
import androidx.fragment.app.viewModels
import com.example.mobile6.databinding.FragmentAppointmentsBinding
import com.example.mobile6.ui.base.BaseFragment
import dagger.hilt.android.AndroidEntryPoint

@AndroidEntryPoint
class AppointmentsFragment : BaseFragment<FragmentAppointmentsBinding>() {
    private val viewModel: AppointmentsViewModel by viewModels()

    override val bindingInflater: (LayoutInflater, ViewGroup?) -> FragmentAppointmentsBinding
        get() = { layoutInflater, container ->
            FragmentAppointmentsBinding.inflate(layoutInflater, container, false)
        }

    override fun initViews() {

    }

    override fun initObservers() {

    }
}